src/cli.rs

```diff
@@ -64,7 +64,15 @@ pub struct Config {
 
     /// If specified, will skip these subreddits
     #[clap(long, env = "SHREDDIT_SKIP_SUBREDDITS")]
-    pub skip_subreddits: Option<SubredditSet>,
+    pub skip_subreddits: Option<ShredditSet>,
+
+    /// If specified, will skip comments and saved comments with listed ids
+    #[clap(long, env = "SHREDDIT_SKIP_COMMENT_IDS")]
+    pub skip_comment_ids: Option<ShredditSet>,
+
+    /// If specified, will skip posts and saved posts with listed ids
+    #[clap(long, env = "SHREDDIT_SKIP_POST_IDS")]
+    pub skip_post_ids: Option<ShredditSet>,
```

Comment:
i'm wonderinf if it might be more ideal to have `HashSet<String>` used directly instead of having a newtype at all. what was the reason for the newtype in the first place?

Reply:
Clap doesn't seem to support HashSet<String> so I had to somehow convert it to this. There might be a better solution than mine, wdyt?

Reply:
ah i see, makes sense!